Classes/Console/Mvc/Cli/CommandConfiguration.php

```diff
@@ -144,6 +144,10 @@ public function addCommandControllerCommands(array $commandControllers): array
         $addedCommandDefinitions = $this->getCommandDefinitionsForCommands(self::unifyCommandConfiguration(['controllers' => $commandControllers], ''));
         $this->commandDefinitions = array_replace($this->commandDefinitions, $addedCommandDefinitions);
 
+        if (!empty($addedCommandDefinitions)) {
+            trigger_error('Registering commands via $GLOBALS[\'TYPO3_CONF_VARS\'][\'SC_OPTIONS\'][\'extbase\'][\'commandControllers\'] is deprecated. Use Configuration/Commands instead.', E_USER_DEPRECATED);
```

Comment:
This should be `Configuration/Commands.php` also should mention that the commands need to be implemented as Symfony commands:

    Registering commands via $GLOBALS[\'TYPO3_CONF_VARS\'][\'SC_OPTIONS\'][\'extbase\'][\'commandControllers\'] is deprecated. Use Symfony commands and Configuration/Commands.php instead.